bin/check_key_order.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+CORRECT_ORDER = %w[uuid reimplements description comments scenarios property input expected]
+
+require 'json'
+
+def find_tests(json)
+  if json['cases']
+    json['cases'].flat_map {|cases| find_tests(cases)}
+  elsif json['uuid']
+    [json]
+  end
+end
+
+exit_code = 0
+
+Dir.glob('exercises/*/canonical-data.json').each do |path|
```

Comment:
this has made it so that this can only be run from the root of the `problem-specifications` repo. If you want to instead make it runnable from anywhere, you should express the path relative from `bin`. 

```suggestion
Dir.glob("#{__dir__}/../exercises/*/canonical-data.json").each do |path|
```

But, I do not actually expect wanting to run it from anywhere else to be something that is often desired, so I'm Approving (in the GitHub sense of the word) anyway regardless of whether this change is made.

Reply:
I've applied the change, with a minor change to allow the printed path to be slightly clearer